src/a_star_search/core.clj

```diff
@@ -1,24 +1,44 @@
 (ns a-star-search.core
   (:require [search.algorithms :as alg]
-            [problems.n-puzzle :as np])
+            [problems.n-puzzle :as np]
+            [problems.sudoku :as su :refer [b]])
   (:gen-class))
 
 (defn print-results [result path costs goal-state]
-  (println (str "We explored " (count result) " states."))
-  (println "The path to the solution is:")
-  (doseq [b (map :board path)]
-    (println b))
-  (when costs
-    (println "The path has" (count path) "steps and its cost is" (costs goal-state))))
+  (do
+    (println "GOAAAAAAAAAAAAAAAAAALLLLLLLLLLLL!")
+    (loop [lst (partition 9 goal-state)]
+      (if (not-empty lst)
+        (do (println (first lst))
+          (recur (rest lst)))))
+    (println (count goal-state) "\n")
+    (println "Don't try to print the path.")))
```

Comment:
This is a complete cop out. All `print-results` now does is print the `goal-state` without paying any attention to the value(s) returned by the search algorithm. So the search algorithm could return `"I'm silly and do nothing"`, and you'd still print `"GOAAAAAAAAAAAAAAAAAALLLLLLLLLLLL!"` and claim to have solved the problem. This is not cool on a lot of levels.